src/Microsoft.Azure.CosmosRepository/Repositories/DefaultRepository.Delete.cs

```diff
@@ -3,6 +3,8 @@
 
 
 // ReSharper disable once CheckNamespace
+using Newtonsoft.Json.Linq;

```

Comment:
I don't think we want `Newtonsoft.Json.*` here at all. We've been trying to remove a single serializer choice, and instead use the `CosmosSerializer` abstraction.

Reply:
You're right, ghost import